src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const thisCalculator = {
```

Comment:
It would be better to name variable as calculator

[NAMING] - Use proper names for variables. If function was called makeFruit, it should return fruit

Reply:
calculator is a better name, but thisCalculator is also a descriptive name, so it's ok for me

Reply:
Tnx guys for review)